app/services/versioning/revision_updater/file_attachment.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Versioning
+  class RevisionUpdater
+    module FileAttachment
+      def update_file_attachment(attachment_revision)
+        revisions = other_file_attachments(attachment_revision) + [attachment_revision]
+        assign(file_attachment_revisions: revisions)
+      end
+
+    private
+
+      def other_file_attachments(attachment_revision)
+        revision.file_attachment_revisions.reject do |revision|
```

Comment:
Revision being shadowed on the end of the line is confusing, maybe the block parameter should be made into an acronym `far`, this is what's currently done in the image mixin for revision updater - `revision_updater/image.rb`.

Reply:
Sorted